home/app_manager.py

```diff
@@ -410,7 +410,9 @@ def _show_msg_success(self, msg):
 
     def _show_msg_failure(self, msg):
         """Show a message indicating failure to execute a requested operation."""
-        self.install_info.show_temporary_message(HTML_MSG_FAILURE.format(msg))
+        # DH: Setting the value directly so that the error message
+        # does not disappear
+        self.install_info.value = HTML_MSG_FAILURE.format(msg)
```

Comment:
I hate when error messages just disappear. That's not a good design.

Reply:
Just a quick question on this, if change the version and retrigger the installation, the output will be wiped and logged from scratch or append to the existing content? 
For the increase in the output panel part, I 100% agree.